regression/cbmc/trace-values/trace-values.desc

```diff
@@ -12,7 +12,7 @@ trace-values.c
 ^  null\$object=6 .*$
 ^  junk\$object=7 .*$
 ^  dynamic_object1\[1.*\]=8 .*$
-^  my_nested\[1.*\]={ .f=0, .array={ 0, 4, 0 } } .*$
+^  my_nested\[1.*\]=\{ .f=0, .array=\{ 0, 4, 0 \} \} .*$
```

Comment:
Ok (I was actually kind-of surprised this passed back when the test was added) - but really this doesn't need to be in this PR and should be a PR of its own instead? And what made you notice the problem? What works now that didn't work before?